crafters/image/ImageResizer/tests/test_imageresizer.py

```diff
@@ -52,22 +55,3 @@ def test_resize_gray():
     img_array = create_random_gray_img_array(img_width, img_height)
```

Comment:
why do we need this second part of the test? can't it be tested on the previous crafted?